tests/cts/test_sparql_resolver/test_sleepy_cat.py

```diff
@@ -0,0 +1,119 @@
+from tests.cts.test_resolver import TestXMLFolderResolverBehindTheScene, TextXMLFolderResolver, TextXMLFolderResolverDispatcher
+from capitains_nautilus.cts.resolver import SleepyCatCTSResolver
+from capitains_nautilus.collections.sparql import clear_graph, \
+    generate_sleepy_cat_graph
+from MyCapytain.common.constants import set_graph
+from ..config import sleepy_cat_address
+from ...sparql_class import Sparql
+import shutil
+
+
+class _Parser(Sparql):
+    RESOLVER_CLASS = SleepyCatCTSResolver
+
+
+def remove_sleepy_cat(address):
+    shutil.rmtree(address, ignore_errors=True)
+
+
+class TestSparqlSleepyCatBasedResolverDispatcher(_Parser, TextXMLFolderResolverDispatcher):
```

Comment:
`TextXMLFolderResolverDispatcher` here should probably be `TextXmlFolderResolverDispatcher` instead.